services/src/main/java/org/keycloak/broker/jwtauthorizationgrant/JWTAuthorizationGrantConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.keycloak.broker.jwtauthorizationgrant;
+
+
+
+public interface JWTAuthorizationGrantConfig {
```

Comment:
Maybe I would use this as a class directly instead of `JWTAuthorizationGrantIdentityProviderConfig`. Similar to what I'm doing in https://github.com/rmartinc/keycloak/tree/pr-44176. But not totally sure, so just a comment.